tests/resources/generateCerts.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+# stop script on error
+set -e
+
+mkdir certGeneration
+cp unittests.key certGeneration/unittests.key
+cp unittests.conf certGeneration/unittests.conf
+cp ca_root.cnf certGeneration/ca_root.cnf
+cd certGeneration
+
+touch index.txt
+echo 1000 > serial
+
+openssl genrsa -out ca_root.key 2048
```

Comment:
trivial/optional: we don't *need* to regenerate the keys, just the certs, but you could leave this in as a comment as to how it was made